src/Store.php

```diff
@@ -37,6 +37,9 @@ class Store
 
   protected $databasePath = "";
 
+  /** @var bool $prettyPrint State of using JSON_PRETTY_PRINT for writing the JSON Files */
+  protected bool $prettyPrint = false;
```

Comment:
Type declarations are supported in PHP >= 7.4

Reply:
Done